backend/s3/file.go

```diff
@@ -29,6 +31,14 @@ type File struct {
 	writeBuffer *bytes.Buffer
 }
 
+type S3Downloader interface {
```

Comment:
For completeness, this should include godoc comments.

Reply:
@seeroush 
Something like this work?

**// Interface needed to mock S3 Downloader data access object in integration tests**

Reply:
As a side note as part of the whole refactor file options things. I do think it could be beneficial to use dependency injection for the s3 downloader to avoid having to do some of the abstractions needed to mock things out with the current constraints.

Reply:
Looks like John ran the integration tests and things passed.